README.md

```diff
@@ -1 +1,40 @@
-# tutorial-linters
\ No newline at end of file
+# tutorial-linters
+# Build With
+ HTML
+ CSS
+# Getting Started
```

Comment:
### Optional 

- [ ] Kindly add a description about your project, this helps give information about the project